.env

```diff
@@ -0,0 +1,2 @@
+MYSQLDB_DATABASE=book_store
+MYSQLDB_ROOT_PASSWORD=4321
```

Comment:
Don’t push `.env` file to GH. Add it to .gitignore. You should create a file `.env.sample` with variables without values

Reply:
fix comment 

`git rm --cached .env` - .env should never be in version control, add `.env.template` instead with actual values removed - otherwise we would need to search for all env var throughout the project to start it